client/src/transaction/api/storage.rs

```diff
@@ -17,6 +18,12 @@ pub struct ApplyRequest {
     pub writelog: WriteLog,
 }
 
+grpc_method!(
+    METHOD_APPLY,
+    "/oasis-core.Storage/Apply",
+    ApplyRequest,
+    Value
```

Comment:
So this being `Value` is not very useful. Can you add a receipt type, even if it is just a value internally, e.g.:
```rust
// XXX: This should be an actual receipt once somebody needs it from Rust.
/// A storage receipt.
type Receipt = Value;
```
And then the result type should be `Vec<Receipt>`.